### ISSUE TITLE
Add a initial page option

### ISSUE THREAD

**Issue by SteffenHummel** [meinto/react-native-viewpager-carousel#13]
Please add a key function to pagerdata and a initial page option

for example:
```
const data = {
  title: "Demo"
  key: "DemoKey"
  component: DemoComponent,
}

```

 ```
<ViewPager
  data={this.data}
  initialPage={{key:"DemoKey}}
  ....
        />
```

**Comment by meinto** [comment]
has been implemented in the way described above

**Comment by meinto** [comment]
is available in version 0.1.13-rc.1
